prepare-vms/workshopctl

```diff
@@ -22,6 +22,8 @@ DEPENDENCIES="
     jq
     pssh
     wkhtmltopdf
+    jinja2
```

Comment:
Uh?

Reply:
```
$ grep jinja2 prepare-vms/lib/ips-txt-to-html.py
import jinja2
template = jinja2.Template(open(template_file_name).read())
```
It's called when generating cards :woman_shrugging:

Reply:
Yes but, does Jinja install an executable called `jinja2`? I _think_ that's what `DEPENDENCIES=` is used for ...

Reply:
(At least on my machine, I don't have a `jinja2` executable, so this check will fail all the time, same for `pdfkit`)